types/index.d.ts

```diff
@@ -32,11 +32,13 @@ export interface ErrorDetails {
     schemaPath: string;
 }
 
+/* eslint-disable no-unused-vars */
```

Comment:
I have no idea why TypeScript is unhappy about these value not being used, considering that these are exported typings, but it is.